pkg/doc/jose/encrypter.go

```diff
@@ -47,7 +46,7 @@ type Encrypter interface {
 
 type encPrimitiveFunc func(*keyset.Handle) (api.CompositeEncrypt, error)
 
-// JWEEncrypt is responsible for encrypting a plaintext and its AAD into a protected JWE and decrypting it
+// JWEEncrypt is responsible for encrypting a plaintext and its AAD into a protected JWE and decrypting it.
```

Comment:
@Baha-sk fyi

Reply:
Below changes in `NewJWEEncrypt` function look ok.